src/ORM/BehaviorRegistry.php

```diff
@@ -58,9 +58,22 @@ class BehaviorRegistry extends ObjectRegistry
     /**
      * Constructor
      *
-     * @param \Cake\ORM\Table $table The table this registry is attached to
+     * @param \Cake\ORM\Table $table The table this registry is attached to.
      */
-    public function __construct(Table $table)
+    public function __construct(Table $table = null)
```

Comment:
@robertpustulka No, only the constructor.

@ADmad I think that would require a closer look, I'm sure behaviors asume a lot about the type of object that gets passed.